### ISSUE TITLE
onOpen callback is called too many times

### ISSUE THREAD

**Issue by padupuy** [robinparisi/tingle#22]
Here is the code : 
```
var modal = new tingle.modal({
      onOpen:function () {
        console.log('onOpen callback is called');
      }
    });
```

When you close and reopen the modal several times 
```
modal.open();
modal.close();
modal.open();
modal.close();
modal.open();
modal.close();
modal.open();
```
You can see that the onOpen callback is called too many times.
<img width="216" alt="capture d ecran 2016-08-09 a 12 18 15" src="https://cloud.githubusercontent.com/assets/715255/17513123/784f7524-5e2b-11e6-9d48-9dd811bef239.png">

When I look at the code  (line 94)
https://github.com/robinparisi/tingle/blob/master/src/tingle.js#L94
there is an `addEventListener` which is attached to the modal every time the onOpen callback is called.  
I think this is the origin of the issue.

**Comment by robinparisi** [comment]
Hi,

Good catch! I've just published a patch on version 0.7.3.

Thanks for the detailed issue 😄

**Comment by padupuy** [comment]
Thank you @robinparisi for your quick support
